src/pages/account/organization/new/banner/Banner.component.js

```diff
@@ -0,0 +1,22 @@
+import { BannerContent, BannerImage } from './banner.styled';
+import data from './banner.data';
+import { Link } from '@tidb-community/ui';
+
+const Banner = () => {
+  return <BannerContent>
+    <BannerImage />
+    <p>
+      {data.leading}
+    </p>
+    <ul>
+      {data.benefits.map((benefit, i) => (
+        <li key={i}>{benefit}</li>
+      ))}
+    </ul>
+    <p>
+      <Link href={data.more.link}>{data.more.title}</Link>
+    </p>
+  </BannerContent>;
+};
```

Comment:
Why the banner component is not formatted by Prettier? Is there anything wrong with the [list-stage configs](https://github.com/pingcap/tug-website/blob/main/lint-staged.config.js) or your IDE?